app/services/weets.js

```diff
@@ -0,0 +1,14 @@
+const api = require('../../config/axios');
+
+const fetchWeet = () =>
+  api
+    .get('/random')
+    .then(({ data }) => data)
+    .catch(error => {
+      console.error(error);
+      return error;
```

Comment:
here wrap the error in a `Promise.reject(error)`, this is because if you return it like this `return error` it is not going to work as an exception anymore, it would be just an error object